lib/utils/replace-info.js

```diff
@@ -1,5 +1,3 @@
-const URL = require('url')
-
```

Comment:
```sugesstion
const URL = require('url').URL

```

This is backwards compatible further than 10.x and matches the style of other uses in the code base (and related modules)

Reply:
You're right, Node.js v6 is listed as a supported engine. It should be noted that WHATWG URL is available as of 6.13.0, not 6.2.0. Probably not a big deal, just wanted to point it out.

https://github.com/npm/cli/blob/ad8fe71a4f8c6d717375a6c699aaa00f11521adf/package.json#L310-L312